app_test.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"testing"
+
+	termbox "github.com/nsf/termbox-go"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInitNavigationKeys(t *testing.T) {
```

Comment:
Don't mind this test, it was improved in a further commit [b162048](https://github.com/taylorskalyo/goreader/pull/12/commits/b162048764ee4ebd1acd2b55acffa0af07711f5b).